extensions/wikia/Blogs/BlogsHelper.class.php

```diff
@@ -171,4 +173,14 @@ public static function onAbortMove( Title $sourceTitle, Title $targetTitle, User
 
 		return true;
 	}
+
+	public static function onAfterPageHeaderButtons( &$buttons ) {
+		if ( F::app()->wg->Title->getNamespace() == NS_BLOG_LISTING ) {
```

Comment:
can we pass $title as hook parameter? if not then why not to use \RequestContext::main()->getTitle()?